srml/session/src/lib.rs

```diff
@@ -80,39 +70,38 @@ macro_rules! impl_session_change {
 
 for_each_tuple!(impl_session_change);
 
-
 pub trait Trait: timestamp::Trait {
-	type ConvertAccountIdToSessionKey: Convert<Self::AccountId, Self::SessionKey>;
-	type OnSessionChange: OnSessionChange<Self::Moment>;
-	type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
+    type ConvertAccountIdToSessionKey: Convert<Self::AccountId, Self::SessionKey>;
+    type OnSessionChange: OnSessionChange<Self::Moment>;
+    type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
 }
 
 decl_module! {
-	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
-		fn deposit_event<T>() = default;
-
-		/// Sets the session key of `_validator` to `_key`. This doesn't take effect until the next
-		/// session.
-		fn set_key(origin, key: T::SessionKey) {
-			let who = ensure_signed(origin)?;
-			// set new value for next session
-			<NextKeyFor<T>>::insert(who, key);
-		}
-
-		/// Set a new session length. Won't kick in until the next session change (at current length).
-		fn set_length(new: <T::BlockNumber as HasCompact>::Type) {
-			<NextSessionLength<T>>::put(new.into());
-		}
-
-		/// Forces a new session.
-		fn force_new_session(apply_rewards: bool) -> Result {
-			Self::apply_force_new_session(apply_rewards)
-		}
-
-		fn on_finalise(n: T::BlockNumber) {
-			Self::check_rotate_session(n);
-		}
-	}
+    pub struct Module<T: Trait> for enum Call where origin: T::Origin {
```

Comment:
Tabs, not spaces. Please set your editor accordingly.

Reply:
Should be fixed now.